providers/nitro/nitro-helper/src/main.rs

```diff
@@ -18,6 +26,43 @@ use tracing::Level;
     about = "helper (proxies etc.) for nitro enclave execution"
 )]
 enum TmkmsLight {
+    #[structopt(flatten)]
+    Helper(CommandHelper),
+    Enclave(CommandEnclave),
+}
+
+/// enclave sub-commands
+#[derive(Debug, StructOpt)]
+enum CommandEnclave {
+    #[structopt(name = "info", about = "get tmkms info")]
+    Info,
+    #[structopt(name = "run", about = "run enclave")]
+    RunEnclave {
```

Comment:
quite useful to have these "shorthand" commands, so one doesn't need to repeat the args (it'd just be automatically read  or taken from a config file) 👍